groovy/JenkinsUtils.groovy

```diff
@@ -9,6 +9,20 @@ void runTests(propertyFile) {
   }
 }
 
+void runTestsDocker(propertyFile) {
+  sh "cp devops/f4a/" + propertyFile + " f4a/FitNesseForAppian/fitnesse-automation.properties"
+  startWebDriverContainer()
+  dir("f4a/FitNesseForAppian") {
+    sh script: "bash ./runFitNesseTest.sh"
+  }
+}
+
+void startWebDriverContainer() {
+  sh "docker-compose -f docker/docker-compose.yml up firefox &"
+  sh "docker-compose -f docker/docker-compose.yml up chrome &"
```

Comment:
not specifying the service would bring up both of them automatically. Do you think that specifying them individually  makes it significantly easier for them to modify for their own purposes?

Reply:
Nope, good catch